### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ananthavijay**
This PR fixes this issue : https://github.com/JBossOutreach/gci/issues/123

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Have you tested it locally?
Also, send PR from one account only.

**Comment by Ananthavijay** [comment]
oh kk. How to test it?

**Comment by Ananthavijay** [comment]
It is not working??

**Comment by Ananthavijay** [comment]
Its not working? I dont know why. @aashutoshrathi 
You check it :+1:

**Comment by Ananthavijay** [comment]
and @aashutoshrathi I think we should update the api  link here also : https://github.com/JBossOutreach/gci/blob/master/scripts/cibuild

**Comment by aashutoshrathi** [comment]
No need

**Comment by Ananthavijay** [comment]
oh kk

**Comment by Ananthavijay** [comment]
Hi @aashutoshrathi so is the issue fixed or not?

**Comment by Ananthavijay** [comment]
@aashutoshrathi ??

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# jboss-outreach.github.io/gci
+# JbossOutreach.github.io/gci
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=jboss-outreach/gci&amp;utm_campaign=Badge_Grade) 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JbossOutreach/gci&amp;utm_campaign=Badge_Grade) 
 [![Join the chat at https://gitter.im/jboss-outreach/gci](https://badges.gitter.im/jboss-outreach/gci.svg)](https://gitter.im/jboss-outreach/gci?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
-[![Build Status](https://travis-ci.org/jboss-outreach/gci.svg?branch=master)](https://travis-ci.org/jboss-outreach/gci)
+[![Build Status](https://travis-ci.org/JbossOutreach/gci.svg?branch=master)](https://travis-ci.org/JbssOutreach/gci)
```
Comment (aashutoshrathi) [inline README.md]: At least spell it right 😟

Comment (aashutoshrathi) [reply README.md]: Spell exact `JBossOutreach` don't even change the case, Please.

**Comment by Ananthavijay** [comment]
oh kk

**Comment by Ananthavijay** [comment]
Is this okay? @aashutoshrathi

**Comment by Ananthavijay** [comment]
?? @aashutoshrathi

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# jboss-outreach.github.io/gci
+# JbossOutreach.github.io/gci
```
Comment (aashutoshrathi) [inline README.md]: Change title to `gci.jbossoutreach.org`

**Comment by Ananthavijay** [comment]
@aashutoshrathi done.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# jboss-outreach.github.io/gci
+# gci.jbossoutreach.org
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=jboss-outreach/gci&amp;utm_campaign=Badge_Grade) 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JbossOutreach/gci&amp;utm_campaign=Badge_Grade) 
 [![Join the chat at https://gitter.im/jboss-outreach/gci](https://badges.gitter.im/jboss-outreach/gci.svg)](https://gitter.im/jboss-outreach/gci?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
-[![Build Status](https://travis-ci.org/jboss-outreach/gci.svg?branch=master)](https://travis-ci.org/jboss-outreach/gci)
+[![Build Status](https://travis-ci.org/JbossOutreach/gci.svg?branch=master)](https://travis-ci.org/JbossOutreach/gci)
```
Comment (aashutoshrathi) [inline README.md]: Replace it by this:
```
[![Build Status](https://travis-ci.org/JBossOutreach/gci.jbossoutreach.org.svg?branch=master)](https://travis-ci.org/JBossOutreach/gci.jbossoutreach.org)
```

[Build Status](https://travis-ci.org/JBossOutreach/gci.jbossoutreach.org)

**Comment by Ananthavijay** [comment]
done @aashutoshrathi :+1:

**Comment by aashutoshrathi** [comment]
Finally done!

**Comment by aashutoshrathi** [comment]
Can you squash your commits?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# jboss-outreach.github.io/gci
+# gci.jbossoutreach.org
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=jboss-outreach/gci&amp;utm_campaign=Badge_Grade) 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/61d5e178fd0f4212a3479528e40de20e)](https://www.codacy.com/app/garg-anuj/gci?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JbossOutreach/gci&amp;utm_campaign=Badge_Grade) 
 [![Join the chat at https://gitter.im/jboss-outreach/gci](https://badges.gitter.im/jboss-outreach/gci.svg)](https://gitter.im/jboss-outreach/gci?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
-[![Build Status](https://travis-ci.org/jboss-outreach/gci.svg?branch=master)](https://travis-ci.org/jboss-outreach/gci)
+[![Build Status](https://travis-ci.org/JBossOutreach/gci.jbossoutreach.org.svg?branch=master)](https://travis-ci.org/JBossOutreach/gci.jbossoutreach.org)
 
 The student site for GCI 2017 Students.
```
Comment (aashutoshrathi) [inline README.md]: Also, I would suggest that you change all `2017` to `2018`

Comment (Ananthavijay) [reply README.md]: yeah sure @aashutoshrathi

**Pull request merged**